test/test-max-data-size.ts

```diff
@@ -75,32 +73,26 @@ describe('maxDataSize', function() {
       location: breakpointInFoo.location
     } as stackdriver.Breakpoint;
     // TODO: Determine how to remove this cast to any.
-    api.set(bp, function(err1) {
+    api.set(bp, (err1) => {
       assert.ifError(err1);
-      // TODO: Determine how to remove this cast to any.
-      api.wait(bp, function(err2?: Error) {
+      api.wait(bp, (err2?: Error) => {
         assert.ifError(err2);
-        // TODO: Determine how to remove this cast to any.
-        assert(bp.variableTable.some(function(v) {
-          // TODO: Handle the case when v is undefined
-          // TODO: Handle the case when v.status is undefined
```

Comment:
Similar question about these TODOs.

Reply:
For tests, if the value of a variable is `undefined`, accessing an attribute on that variable will cause an Error that will fail the test.  Thus, I think it is safe to use the syntax `someVar!.someAttribute` in tests.